Makefile

```diff
@@ -245,7 +245,7 @@ dist:
 deb: contrib/debian/*
 deb: dist
 	# Rename dist tarball to .orig.tar.gz as the source format version "3.0 (quilt)" requires
-	mv /tmp/aziot-identity-service-$(PACKAGE_VERSION).tar.gz /tmp/aziot-identity-service_$(PACKAGE_VERSION).orig.tar.gz
+	mv /tmp/aziot-identity-service-$(PACKAGE_VERSION).tar.gz /tmp/aziot-identity-service_$(PACKAGE_VERSION)~$(PACKAGE_RELEASE).orig.tar.gz
```

Comment:
This is also wrong. The source tarball does not contain the release identififer either.

Reply:
Fixed. The rpm tarball below does contain the revision though, should I change that so that it doesn't or leave as is?

Reply:
It's a matter of conventions. Debian requires the release ident to not be there (IIRC). rpm doesn't care either way because you tell it what the tarball name is, and the convention is to include the release ident.